src/components/content-tab-assistant.tsx

```diff
@@ -13,6 +14,10 @@ import Button from './button';
 import { AssistantIcon } from './icons/assistant';
 import { MenuIcon } from './icons/menu';
 
+interface SiteDetails {
```

Comment:
It seems odd to redeclare this [existing interface](https://github.com/Automattic/studio/blob/a7af92333a6bc862533a455dd4dd1884429ace88/src/ipc-types.d.ts#L39). Is there a reason to redeclare it here?